package.json

```diff
@@ -9,17 +9,17 @@
   "license": "ISC",
   "repository": "your-repository-name",
   "scripts": {
+    "dev": "nodemon --exec \"npm start\"",
```

Comment:
👍 Props for thinking about the use case where you start on Glitch but run it locally later.

Reply:
Thank you. 😄